ee/codegen/src/generators/nodes/conditional-node.ts

```diff
@@ -24,9 +24,13 @@ export class ConditionalNode extends BaseSingleFileNode<
   protected getNodeClassBodyStatements(): AstNode[] {
     const statements: AstNode[] = [];
 
-    const inputVarIdsByRuleId = new Map<string, string>();
-    const constantValuesByRuleIds = new Map<string, VellumValue>();
-    this.constructMapsByRuleIds(inputVarIdsByRuleId, constantValuesByRuleIds);
+    const inputFieldKeysByRuleId = new Map<string, string>();
+    const valueInputKeysByRuleId = new Map<string, string>();
```

Comment:
I ended up needing these two maps because the nodeInputsByKey had keys as the Map.key. I wanted an easy way to just access the values so I built a map from (ruleId -> input.key) and then use that key to search up the value instead of doing a reverse lookup which would have gotten kind of messy but can change on this approach. I am 1/5 on changing this for something better